server/src/serializers/MovieSerializer.js

```diff
@@ -26,9 +32,11 @@ class MovieSerializer {
     const serializedReviews = reviews.map((review) => {
       return ReviewSerializer.cleanReview(review);
     });
-
     serializedMovie.reviews = serializedReviews;
 
+    let averageRating = await movie.$relatedQuery("reviews").avg("rating");
+    serializedMovie.averageRating = parseFloat(averageRating[0].avg).toFixed(1);
```

Comment:
even more reason why this needs to be a model method instead of logic repeated in multiple serializers